WordPress/src/main/res/values/strings.xml

```diff
@@ -1623,9 +1623,8 @@
     <!-- reader -->
     <string name="reader">Reader</string>
     <string name="reader_filter_cta">Filter</string>
-    <string name="reader_filter_sites_title">Sites</string>
-    <string name="reader_filter_tags_title">Topics</string>
-    <string name="reader_filter_main_title">Following</string>
+    <string name="reader_filter_sites_title">Filter by blog</string>
```

Comment:
If I remember correctly, the script (or maybe GlotPress) is smart enough to see the changes and mark the Strings as "untranslated" in the platform. cc @oguzkocer or @AliSoftware to confirm that though.

Reply:
We should always treat strings as immutable—once you created a key with a value, you should avoid changing its value later on, especially if the first value already made it to a previous release.

This is because otherwise once this change hits trunk it will be sent to GlotPress and the new copy retranslated… but if we then pull the translations from GlotPress into the `release/*` branch before submitting to Google then that change of the copy would end up making its way to the release branch… despite being a change tied to something in `trunk`.

In summary modifying the value of an existing key will cause a discrepancy in the `release/*` branch, between the codebase (which will expect the old copy  as that's what it was when code was frozen) and the translations (updated with the new copy when we'll pull the latest from GlotPress)

TL;DR: prefer using a brand new key.

Reply:
Thanks, @AliSoftware 

I'll change the string keys with new content to brand new ones.

Reply:
Thanks @AliSoftware for the answer, that makes sense!